components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/authz/validators/AbstractResponseTypeRequestValidator.java

```diff
@@ -199,7 +199,10 @@ private OAuth2ClientValidationResponseDTO validateCallBack(String clientId, Stri
             validationResponseDTO.setValidClient(false);
             validationResponseDTO.setErrorCode(OAuth2ErrorCodes.UNAUTHORIZED_CLIENT);
             validationResponseDTO
-             .setErrorMsg("The authenticated client is not authorized to use this authorization grant type");
+             .setErrorMsg("not.authorized.for.authorization.grant.type");
+            if (log.isDebugEnabled()) {
```

Comment:
Redundant debug log as the same information is available at https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1901/files#diff-a19d23e58562e03b960b1b8590e9e8d512fe5e820c76d3011d3e81cf518fd2f6R184